tests/utils/test_names.py

```diff
@@ -43,6 +43,20 @@ def test_normalize_name():
     assert normalize_name("Dupont, J.-R.") == "dupont j r"
     assert normalize_name("Dupont") == "dupont"
     assert normalize_name("Dupont J.R.") == "dupont j r"
+    assert normalize_name("von und zu Hohenstein, F.") == "hohenstein f"
+    assert normalize_name("von und zu Hohenstein, F.",
+                          drop_common_affixes=False) == "vonundzuhohenstein f"
+    assert normalize_name("Jakub, Ibrahim ibn") == "jakub ibrahim ibn"
+    assert normalize_name("o'Neill, Jack") == "neill jack"
```

Comment:
That's why it is an argument to the function. Why I was implementing blocking algorithm I got higher recall when I dropped all the affixes. However, the strategy was different back then (double metaphone, moving signatures with multiple surnames to correct blocks, etc.) It might be worth to check the score after this change.